### ISSUE TITLE
Deal with non-CORS URLs

### ISSUE THREAD

**Issue by gavinr** [gavinr/geojson-viewer#1]
If the url is not sending a `CORS: *` header, then we cannot map it. Maybe think about using a public proxy for these urls or some other solution.

At least give an error explaining the situation.

**Comment by acdpsn** [comment]
Can I work on this? It sounds simple enough to just implement an error popup.

**Comment by gavinr** [comment]
@akim8 yes that would be great - basically if a URL that does not have a cors header is used, we should show at the very minimum a modal popup explaining the situation. Thanks!
